incubator/hnc/cmd/manager/main.go

```diff
@@ -135,7 +137,9 @@ func main() {
 	if debugLogs {
 		logLevel = zapcore.DebugLevel
 	}
-	log := zap.New(zap.Level(logLevel), zap.StacktraceLevel(zapcore.PanicLevel))
+	rlog := zap.NewRaw(zap.Level(logLevel), zap.StacktraceLevel(zapcore.PanicLevel))
+	stdzap.RedirectStdLogAt(rlog, zapcore.ErrorLevel)
```

Comment:
Why `ErrorLevel`? Do we know that they'll all be errors?

Reply:
I know the stdlib should only be calling `log` if it has no other way of returning an error, like http.Server before a handler is invoked. I'm assuming the same applies to the other libraries in use

Reply:
Thanks - can you please add that reasoning as a comment in the code? Then I'll approve.